apps/hash-api/src/graph/knowledge/primitive/entity.ts

```diff
@@ -505,7 +504,7 @@ export const getEntityIncomingLinks: ImpureGraphFunction<
       graphResolveDepths: zeroedGraphResolveDepths,
       temporalAxes,
     })
-    .then(({ data }) => mapSubgraph(data) as Subgraph<EntityRootType>);
+    .then(({ data }) => data as Subgraph<EntityRootType>);
```

Comment:
Do we want to change these to be consistent with above?

```suggestion
    .then(({ data: subgraph }) =>
      getRoots(subgraph as Subgraph<EntityRootType>),
    );
```

Reply:
Applied in [Apply suggestions from Code Review](https://github.com/hashintel/hash/pull/2110/commits/460911c284da89861487b718d9a50812321df97f)